packages/synthetix-main/contracts/modules/core/RewardsManagerModule.sol

```diff
@@ -64,6 +61,33 @@ contract RewardsManagerModule is IRewardsManagerModule {
             revert InvalidParameters("distributor", "must be non-zero");
         }
 
+        existingDistribution.distributor = IRewardDistributor(distributor);
+    }
+
+    function setRewardsDistribution(
```

Comment:
Maybe we rename to `distributeRewards` (though this might be confused with a payout). Possible `notify` is the right verb here.